### ISSUE TITLE
timeline breaks with yoothme pro 2.1.1

### ISSUE THREAD

**Issue by dimitrilongo** [forrestkirby/timeline-element#7]
hello

just after update timeline not rendering as expected, for instance no more alternate and line.

Regards

**Comment by forrestkirby** [comment]
Hi dimitrilongo,

I could not reproduce this problem. Have you replaced the files for the Timeline Element by the latest version as well? Make sure you first update the files for the timeline element, then update YOOtheme Pro. Also clear all cache (browser & server).

Regards
Thomas

**Comment by dimitrilongo** [comment]
hi thanks for fast answer,

so this morning i update yootheme first, and just saw this after timeline breaks, so i download master branch, copy files, clear cache etc.

**Comment by forrestkirby** [comment]
I just tested the following steps and did not encounter any issue, neither after updating the template/theme nor the custom element.

- install YOOtheme Pro 2.0.14
- add a child theme with the timeline element
- create a layout with the timeline element
- update to YOOtheme Pro 2.1.1
- update the timeline element

Please provide more details on how to reproduce this problem or an archive of your installation before the update when it was still working. Thanks.

**Comment by dimitrilongo** [comment]
exact same steps this morning,I can send you url of the page  (if yes just tell wich email adress)

**Comment by forrestkirby** [comment]
Do you see any error messages (PHP or JavaScript)?

If you don’t want to publicly share data on GitHub, you can create a new question (with FTP and admin access) in the YOOtheme Support Forum and provide the link to that question here. I will then take a look at the question.

**Comment by dimitrilongo** [comment]
great, i start new thread : https://yootheme.com/support/question/138945

**Comment by forrestkirby** [comment]
Thank you, I could reproduce the issue now (it’s only on WordPress). For the moment, changing [line 16](https://github.com/forrestkirby/timeline-element/blob/master/hd-timeline/element.php#L16) of the `element.php` file to the following should do the trick:

```
$metadata->set('style:builder-hd-timeline', ['href' => Path::get('./css/timeline.css'), 'defer' => true]);
```

The reason seems to be that the behaviour of loading CSS files changed. In YOotheme Pro for WordPress, it’s now possible to load CSS files deferred, but not in Joomla! which results in invalid HTML markup.

I’ll get in touch with YOOtheme for a recommendation and eventually provide an update or a fix.

**Comment by dimitrilongo** [comment]
great thanks again

**Comment by forrestkirby** [comment]
For the sake of documentation: This shall be fixed in YOOtheme Pro for WordPress, so once the fix has been released, the workaround won’t be needed anymore. Thus, I won’t change anything in the Timeline Element repository.

**Comment by forrestkirby** [comment]
Fixed in [YOOtheme Pro 2.2.0-beta.1](https://yootheme.com/support/question/139179)
